GitHub/README.md

```diff
@@ -0,0 +1,44 @@
+
+- updates to dev-only dependencies
+- changes to dev-only tooling (dev tools like the dev server for client-side apps, test runners, CI configuration)
+- changes to the README or documentation
+
+Only changes such as these that are *not* significant may be pushed to master, and even then you may see some of us opening pull requests for such changes if we either wish to expose the changes to our peers for comment or just to prevent potential conflicts on master if a lot of things are currently in flight. Basically: when in doubt, open a pull request!
+
+Note that the following things, which may *seem* insignificant, are subject to code review (and QA):
+
+- updates to production dependencies
+- changes to production build tooling
+- changes to the production static server (for client apps like blip)
+
+### Rebasing
+
+We follow the general guideline of "don't rewrite history." Rebasing to reorder, squash, or otherwise fix commits locally before a branch or a set of commits has been pushed upstream is perfectly acceptable, but once commits are upstream you shouldn't mess with them (by rebasing or, really, by any other means).
+
+The one exception to this is large, long-running feature branches. If you feel a need to clean up a large, long-running branch, the polite thing to do is to ask the other developer(s) you are collaborating with most closely on the work and/or the developer(s) who will be expected to review the code if they mind if you rebase and force push upstream. But again, this is the exception, not the rule.
+
+### Tags
+
+We "cut" tags in two situations, the first far more common than the second:
+
+1. when we think we have a release candidate of an app or service
+1. when we know we *don't* have a release candidate of an app or service, but we want to deploy what we have so far to demo it or make it available for developer use
+
+In the first situation, we create the tag after merging the pull request(s) for the new feature(s) or fix(es) to master. In the second situation, we create the tag on the working feature branch. We use [semantic versioning](http://semver.org/ 'semver.org') for the first type of tag (the uploader is an exception, since the Chrome store does not support patch-level version increments), and most of us use the tool [mversion](https://github.com/mikaelbr/mversion 'mversion') to create these tags in a consistent way. You can bump the minor version for an uploader release, for example, with the command `mversion minor -m` and this will bump the version in *both* the `package.json` and the `manifest.json` (for the Chrome store), as well as commit those changes, named the commit with the new version number, and create the git tag. We typically distinguish the second type of tag from the first to make it very clear that the "demo" tag is not a release candidate, either by adding a post-fix (e.g., `v0.12.0-alpha`) or by using a named tag instead of a semver-based tag (e.g., `clinic-accts-beta`).
```

Comment:
I think semver tags are best for both cases.  For non-release candidates I would *always* add a post-fix to the version.  Also, might want to point out that we use the "v" prefix on the version.  I don't think that is specifically part of semver.